src/box/raft.c

```diff
@@ -301,6 +302,21 @@ box_raft_fence(void)
 	raft_resign(raft);
 }
 
+/**
+ * Resign RAFT leadership and freeze limbo.
```

Comment:
Please describe here why this function is different from `box_raft_fence` above.
For example, this function ignores `election_fencing` setting and doesn't prevent the node from starting new elections.

Reply:
Done.